include/seqan3/argument_parser/argument_parser.hpp

```diff
@@ -826,22 +826,23 @@ class argument_parser
      */
     void verify_identifiers(char const short_id, std::string const & long_id)
     {
-        auto constexpr allowed = is_alnum || is_char<'_'> || is_char<'@'>;
+        std::string const valid_chars{"@_0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"};
+        auto is_allowed = [&valid_chars] (char const c) { return valid_chars.find(c) != std::string::npos; };
```

Comment:
I wanted to avoid the dependence on ranges because if we do not drop GCC9, I would need the seqan3/std/ranges header or copy it to the *sarg* (:smiley_cat:) repo.

Reply:
Yes, that makes perfectly sense and I didn't think as far :)